src/clj/rems/api/blacklist.clj

```diff
@@ -56,6 +58,8 @@
       :body [command BlacklistCommand]
       :return schema/SuccessResponse
       (blacklist/add-user-to-blacklist! (getx-user-id) command)
+      (doseq [cmd (rejecter-bot/reject-all-applications-by (get-in command [:blacklist/user :userid]))]
+        (command/command! cmd))
```

Comment:
Should there be one function in the service layer that does all of this?

Reply:
The problem is that services.command already depends on services.blacklist, so we can't have services.blacklist depend on services.command...

But now that I look at this piece of code I notice that I should check the return value of `command!`

Reply:
Yes it's not atomic. Also it feels a bit strange to have blacklist stuff in command.

Reply:
added the error handling